spec/rubocop/cop/root_cops/factories/factory_name_spec.rb

```diff
@@ -2,12 +2,10 @@
   subject(:cop) { described_class.new }
 
   context "when the factory file name is incorrect" do
-    before do
-      allow(File).to receive(:basename).and_return("user_billing")
-    end
+    let(:factory_file_path) { "/code/server/systems/sample/spec/factories/user_billing.rb" }
 
     it "skips all factory name checks and registers no offenses" do
-      expect_no_offenses(<<-RUBY)
+      expect_no_offenses(<<~RUBY, factory_file_path)
```

Comment:
Nice! Didn't realize this was an option

Reply:
They don't advertise it in the docs, but it's part of a public API, so... 🤞